root/scripts/talker/coach.txt

```diff
@@ -5060,8 +5068,8 @@ Rule SurvivorSpottedCedaCoach
 
 Rule SurvivorSpottedCedaC1M1Coach
 {
-	criteria ConceptPlayerWarnSpecial IsSpecialTypeCeda IsNotCoughing IsCoach IsTalk IsTalkCoach IsWorldTalkCoach IsNotIncapacitated IsNotSaidCedaWarn ismap_c1m1_hotel SawFirstHazmat IsNotSpeakingWeight0
```

Comment:
This means that every survivor would only use the unique c1m1 rule. There wouldn't be context in this regard.

Reply:
This is how the talker is currently in the live version, except I've changed "SaidCedaWarn:1:0" to "SaidCedaWarn:1:20" to allow multiple survivors to comment about the hazmat zombies on c1m1, instead of currently only one.

I felt it was best to save the "fireproof" callouts for c1m2 to reduce the repetition of these lines, and the bombardment of information being dropped don players in the first map. Pre-TLS, this was done by making the "fireproof" lines rare, with only a 5%-10% chance of saying them.